files/react-albus/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "react-albus",
+  "repo": "americanexpress/react-albus",
+  "files": {
+    "basePath": "lib/"
```

Comment:
I don't see the `lib/` folder on the latest tag https://github.com/americanexpress/react-albus/tree/v1.0.1 (perhaps you want to use `npm` as the packageManager?